security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -129,9 +128,10 @@ type sdsservice struct {
 
 	localJWT bool
 
-	jwtPath string
```

Comment:
Please leave the jwtPath - it is a common case to have the file from a JWT, not specific to K8S and not in need of a plugin.

And you can add the new interface as an option - if it is set, it'll be used instead of the jwt path.

Reply:
But only if we confirm that simply adding the gRPC 'per request credential' to the config doesn't eliminate the need to have this credFetcher in the first place.

Reply:
jwtPath is already included in credFetcher. The information is redundant.